api/doc/openapi.yaml

```diff
@@ -975,44 +966,35 @@ components:
         - $ref: '#/components/schemas/GenesisTransaction'
         - $ref: '#/components/schemas/UserTransaction'
         - $ref: '#/components/schemas/BlockMetadataTransaction'
-      discriminator:
```

Comment:
`discriminator` is valid `openapi` field, and was added for `redocly` to render enum, `spotlight/elements` handles `anyOf` properly, so we don't need these anymore.